lnt/server/ui/static/lnt_profile.js

```diff
@@ -1073,10 +1073,12 @@ function FunctionTypeahead(element, options) {
         },
         updater: function(item) {
             // FIXME: the item isn't passed in as json any more, it's
-            // been rendered ("fname,[object Object]"). Lame. Hack around
-            // this by splitting apart the ','-concatenated 2-tuple again.
-            var splitIndex = item.lastIndexOf(',');
-            fname = item.substr(0, splitIndex);
+            // been rendered. Lame. Hack around this by splitting apart
+            // the ','-concatenated 2-tuple again.
+            // tamar: with C++ symbols demangled we have natural ,s in the
```

Comment:
I'm a bit confused as to what the strings coming in here look like.
I'm assuming they don't still look like "fname,[object Object]", as at first sight, the old code would still work by splitting off after the last comma in the string.
But then I see the new code also seems to explicitly remove the last characters, assuming the string ends with ",[object Object]".
I'm most likely misunderstanding something here, but I'm not sure what exactly?

Reply:
ah no, you're right, looking at my local patch, it was at some point changed from
```
-            fname = item.split(',')[0];
+           var splitIndex = item.lastIndexOf(',');
+           fname = item.substr(0, splitIndex);
```
and hadn't noticed when I resolved the conflicts.  So this is indeed not needed.